providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -264,6 +264,7 @@ static int rsa_decrypt(void *vprsactx, unsigned char *out, size_t *outlen,
             /* RSA_PKCS1_WITH_TLS_PADDING */
             if (prsactx->client_version <= 0) {
                 ERR_raise(ERR_LIB_PROV, PROV_R_BAD_TLS_CLIENT_VERSION);
+                OPENSSL_free(tbuf);
```

Comment:
There is one more tbuf leak in ERR_R_INTERNAL_ERROR error case above.

Reply:
Done